### PR TITLE
Multi project

### PR DESCRIPTION
**Pull request by bmblb**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -22,3 +22,99 @@ If you want to specify different port you can call server script directly:
...
+If authorization is successful client will receive a following message:
+
+```json
+{ "command": "login" }
+```
+
+If username or password is incorrect error is returned to the client:
+
+```json
+{ "command": "login", "error": "Wrong username/password" }
```
Comment (isglass) [inline README.md:53]: Perhaps errors should be some kind of code / key, to let client specify actual message to show (also to aid localization a bit)

File: README.md [hunk 2]
```diff
@@ -22,3 +22,99 @@ If you want to specify different port you can call server script directly:
...
+```
+
+If client is not logged in an is trying to access the project data or send updates sever will prevent it
```
Comment (isglass) [inline README.md:56]:
```suggestion
If client is not logged in and is trying to access the project data or send updates sever will prevent it
```

File: README.md [hunk 3]
```diff
@@ -22,3 +22,99 @@ If you want to specify different port you can call server script directly:
...
+
+Response will contain list of project ids client is authorized to access:
+
+```json
+{ "command": "projects", "projects": [1,2] }
+```
+
+### Load project
+
+To start working with a project client should set a `dataset` command specifying id of the project:
```
Comment (isglass) [inline README.md:79]:
```suggestion
To start working with a project client should send a `dataset` command specifying id of the project:
```

File: README.md (hunk 1 above)
Comment (bmblb) [reply README.md:53]: Interesting suggestion. I'm thinking utilizing carefully crafted HTTP codes would be great and it wouldn't require much documentation

**Pull request merged**